translate/pb/pb_decoder_entry.go

```diff
@@ -0,0 +1,76 @@
+package pb
+
+import (
+	"fmt"
+
+	"github.com/arr-ai/arrai/rel"
+	"github.com/arr-ai/arrai/tools"
+	"google.golang.org/protobuf/proto"
+	"google.golang.org/protobuf/reflect/protoreflect"
+	"google.golang.org/protobuf/types/dynamicpb"
+)
+
+// StdProtobufDecoder transforms the protocol buffer message to a tuple.
+var StdProtobufDecoder = rel.NewNativeFunction("decode", func(tupleParam rel.Value) (rel.Value, error) {
+	tuple, isTuple := tupleParam.(rel.Tuple)
+	if !isTuple {
+		return nil, fmt.Errorf("//encoding.proto.decode: tupleParam not tuple")
+	}
+
+	if !tuple.IsTrue() {
+		// the call looks like `/set sysl = //encoding.proto.decode(//encoding.proto.proto, //os.file("sysl.pb"))`
+		return rel.NewNativeFunction("decode$1", func(definition rel.Value) (rel.Value, error) {
+			_, is := tools.ValueAsBytes(definition)
+			if !is {
+				return nil, fmt.Errorf("//encoding.proto.decode: definition not bytes")
+			}
+
+			return rel.NewTuple(rel.NewAttr("fileDescriptor", definition)), nil
+		}), nil
+	}
+
+	// the call looks like `/set decodeSyslPb = //encoding.proto.decode(sysl)`
+	// after `/set sysl = //encoding.proto.decode(//encoding.proto.proto, //os.file("sysl.pb"))`
```

Comment:
Nit: This kind of comment would be useful on the function since it describes usage; it's less useful inside since it doesn't describe the implementation.

Also I think as an example it would be better as a block of plain arr.ai without the `/set`, something like:

```
let sysl = //encoding.proto.decode(//encoding.proto.proto, //os.file("sysl.pb"));
let petshop = //encoding.proto.decode(sysl, 'Module', //os.file("petshop.pb"));
petshop.apps('PetShopApi')
```